### PR TITLE
Upgrade build

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -1,59 +1,42 @@
 // @flow
 import babel from 'rollup-plugin-babel';
-import commonjs from 'rollup-plugin-commonjs';
 import replace from 'rollup-plugin-replace';
-import { uglify } from 'rollup-plugin-uglify';
+import { terser } from 'rollup-plugin-terser';
```
Comment (alexreardon) [inline rollup.config.js:4]: cool

File: .babelrc [hunk 2]
```diff
@@ -1,19 +1,7 @@
...
+    ["@babel/env", { "loose": true }]
   ],
-  "comments": false,
-  "env": {
-    "cjs": {
-      "plugins": [
-        "transform-es2015-modules-commonjs"
-      ]
-    },
-    "test": {
```
Comment (alexreardon) [inline .babelrc:13]: is this no longer needed for jest?

File: package.json [hunk 3]
```diff
@@ -37,22 +37,21 @@
...
-    "babel-cli": "6.26.0",
-    "babel-eslint": "8.2.5",
-    "babel-jest": "^23.2.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-flow": "6.23.0",
+    "@babel/core": "^7.4.5",
+    "@babel/preset-env": "^7.4.5",
+    "@babel/preset-flow": "^7.0.0",
+    "babel-eslint": "10.0.1",
+    "babel-jest": "^24.8.0",
```
Comment (alexreardon) [inline package.json:44]: ah, i am guessing this takes care of the commonjs stuff for jest?

File: .babelrc (hunk 2 above)
Comment (TrySound) [reply .babelrc:13]: babel/env is used without `modules: false` which is internally applied by rollup-plugin-babel. So jest gets transpiled modules.

File: .babelrc [hunk 4]
```diff
@@ -1,19 +1,7 @@
 {
   "presets": [
-    "flow",
-    ["env", { "modules": false, "loose": true }]
+    "@babel/flow",
+    ["@babel/env", { "loose": true }]
```
Comment (TrySound) [inline .babelrc:4]: This includes commonjs by default

File: .babelrc (hunk 2 above)
Comment (TrySound) [reply .babelrc:13]: https://github.com/rollup/rollup-plugin-babel#modules

**Pull request merged**